index.bs

```diff
@@ -140,6 +149,25 @@ The <dfn attribute for="Document">prerendering</dfn> getter steps are to return
 The <dfn attribute for="Document">onprerenderingchange</dfn> attribute is an [=event handler IDL attribute=] corresponding to the <dfn event for="Document">prerenderingchange</dfn> [=event handler event type=].
 
 
+<h2 id="prerender-processing-model">Prerender processing model</h2>
+
+Add a new link type to the "Link types" section of the HTML Standard, replacing the prerender documentation in the Resource Hints standard.
+
+<h3 id="link-rel-prerender">Link type "<dfn data-lt="rel-prerender"><code>prerender</code></dfn>"</h3>
+
+The <a lt="rel-prerender"><code>prerender</code></a> keyword may be used with <a spec=HTML><code>link</code></a> elements. This keyword creates an <a spec=HTML>external resource link</a>. This keyword is <a spec=HTML>body-ok</a>.
+
+To <a lt="process the linked resource" spec=HTML>process this type of linked resource</a> given a <a spec=HTML><code>link</code></a> element <var ignore>el</var>, run these steps:
+
+<div algorithm="prerender-processing-model">
+  1. If |el|'s [=link/href=] attribute's value is the empty string, then return.
+  1. Let |referrer policy| be  the current state of |el|'s [=link/referrerpolicy=] attribute.
+  1. [=parse a URL|Parse=] the [=URL=] given by |el|'s [=link/href=] attribute, relative to |el|'s [=Node/node document=]. If that fails, then return. Otherwise, let |url| be the [=resulting URL record=].
```

Comment:
So, modulepreload does this wrong. It should be "[Parse the URL] given by...". Stating that it's a `[=URL=]` is incorrect since _el_'s `href` attribute is a string, not a URL record. "Parse the URL" would be better named as "Parse the URL string", really...

Reply:
Yikes, I actually got that from https://html.spec.whatwg.org/multipage/semantics.html#default-fetch-and-process-the-linked-resource but yeah that makes sense. I'll fix this and those two instances in HTML.